xss/xss.go

```diff
@@ -128,6 +128,26 @@ func bufferedBodyHandler(w http.ResponseWriter, r *http.Request, safety string)
 	return template.HTML(input), false
 }
 
+func responseHandler(w http.ResponseWriter, r *http.Request, safety string) (template.HTML, bool) {
+	userInput := utils.GetUserInput(r)
+	var ret []byte
```

Comment:
since `ret` is not being returned maybe just replacing lines 138, and 141 with `w.Write([]byte(userInput))` makes more sense? That would simplify some of the code since ret is not used otherwise.